lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -51,6 +51,10 @@ __PACKAGE__->add_columns(
         false_id      => ['0', '-1'],
         default_value => '0',
     },
+    feature_version => {
+        data_type     => 'integer',
+        default_value => 0,
```

Comment:
I disabled the feature tour temporarly, since i dont have a real tour yet. Will do a real tour in the next step, but i thought it would be a good idea to make a merge of the branch possible.